diff -urN aa/gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c
--- aa/gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c	2003-05-02 15:55:39.000000000 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-capplet.c	2004-07-26 14:15:43.635339192 +0200
@@ -183,6 +183,7 @@
 	init_widgets (gpcap);
 
  error:
+ ;
 }
 
 GnomePilotCapplet *
diff -urN aa/gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c
--- aa/gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c	2003-04-22 22:51:06.000000000 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-cdialog.c	2004-07-26 14:16:57.728075384 +0200
@@ -123,6 +123,7 @@
 	gnome_dialog_close_hides (GNOME_DIALOG (priv->dialog), TRUE);
 	
  error:
+ ;
 }
 
 
diff -urN aa/gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c
--- aa/gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c	2003-04-22 23:12:36.000000000 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-ddialog.c	2004-07-26 14:16:32.905848936 +0200
@@ -123,6 +123,7 @@
 	}
 
  error:
+ ;
 }
 
 
diff -urN aa/gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c
--- aa/gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c	2003-05-13 19:29:59.000000000 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-druid.c	2004-07-26 14:18:16.487102192 +0200
@@ -181,6 +181,7 @@
 	init_widgets (gpd);
 
  error:
+ ;
 }
 
 
diff -urN aa/gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c
--- aa/gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c	2003-04-23 17:39:44.000000000 +0200
+++ gnome-pilot-2.0.10/capplet/gnome-pilot-pdialog.c	2004-07-26 14:16:20.664709872 +0200
@@ -144,6 +144,7 @@
 	}
 
  error:
+ ;
 }
 
 
