diff -Nru gnome-pilot-0.1.54/capplet/Makefile.am gnome-pilot-0.1.54.new/capplet/Makefile.am
--- gnome-pilot-0.1.54/capplet/Makefile.am	Sat Jul  1 02:58:49 2000
+++ gnome-pilot-0.1.54.new/capplet/Makefile.am	Sun Oct 29 15:27:10 2000
@@ -63,13 +63,13 @@
 appicon_DATA = gnome-palm.png
 
 install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
-	$(mkinstalldirs) $(pConduitsdir)
-	$(mkinstalldirs) $(gladedir)
-	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(Conduitsdir)/.directory
-	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(pConduitsdir)/.directory
+	$(mkinstalldirs) $(DESTDIR)$(Conduitsdir)
+	$(mkinstalldirs) $(DESTDIR)$(pConduitsdir)
+	$(mkinstalldirs) $(DESTDIR)$(gladedir)
+	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(DESTDIR)$(Conduitsdir)/.directory
+	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(DESTDIR)$(pConduitsdir)/.directory
 
-#	$(mkinstalldirs) $(PalmPilotdir)
-#	$(mkinstalldirs) $(pPalmPilotdir)
-#	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(PalmPilotdir)/.directory
-#	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(pPalmPilotdir)/.directory
+#	$(mkinstalldirs) $(DESTDIR)$(PalmPilotdir)
+#	$(mkinstalldirs) $(DESTDIR)$(pPalmPilotdir)
+#	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(DESTDIR)$(PalmPilotdir)/.directory
+#	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(DESTDIR)$(pPalmPilotdir)/.directory
diff -Nru gnome-pilot-0.1.54/conduits/backup/Makefile.am gnome-pilot-0.1.54.new/conduits/backup/Makefile.am
--- gnome-pilot-0.1.54/conduits/backup/Makefile.am	Wed Jul  5 20:38:57 2000
+++ gnome-pilot-0.1.54.new/conduits/backup/Makefile.am	Sun Oct 29 15:27:51 2000
@@ -54,7 +54,3 @@
 EXTRA_DIST = \
 	$(Conduit_DATA) \
         $(ccenterConduits_DATA)
-
-install-data-local:
-	$(mkinstalldirs) $(Conduitdir)
-	$(mkinstalldirs) $(ccenterConduitsdir)
diff -Nru gnome-pilot-0.1.54/conduits/email/Makefile.am gnome-pilot-0.1.54.new/conduits/email/Makefile.am
--- gnome-pilot-0.1.54/conduits/email/Makefile.am	Sat Jul  1 02:58:51 2000
+++ gnome-pilot-0.1.54.new/conduits/email/Makefile.am	Sun Oct 29 15:28:02 2000
@@ -52,5 +52,3 @@
 	$(ccenterConduits_DATA) \
 	$(Conduit_DATA)
 
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
diff -Nru gnome-pilot-0.1.54/conduits/expense/Makefile.am gnome-pilot-0.1.54.new/conduits/expense/Makefile.am
--- gnome-pilot-0.1.54/conduits/expense/Makefile.am	Sat Jul  1 02:58:51 2000
+++ gnome-pilot-0.1.54.new/conduits/expense/Makefile.am	Sun Oct 29 15:28:13 2000
@@ -50,6 +50,3 @@
 EXTRA_DIST = \
 	$(ccenterConduits_DATA) \
 	$(Conduit_DATA)
-
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
diff -Nru gnome-pilot-0.1.54/conduits/file/Makefile.am gnome-pilot-0.1.54.new/conduits/file/Makefile.am
--- gnome-pilot-0.1.54/conduits/file/Makefile.am	Sat Jul  1 02:58:52 2000
+++ gnome-pilot-0.1.54.new/conduits/file/Makefile.am	Sun Oct 29 15:28:25 2000
@@ -76,11 +76,6 @@
 Mimedir = $(datadir)/mime-info
 Mime_DATA = palm.keys
 
-
-install-data-local:
-	$(mkinstalldirs) $(ccenterConduitsdir)
-	$(mkinstalldirs) $(Conduitdir)
-
 EXTRA_DIST = \
 	$(Conduit_DATA) \
         $(ccenterConduits_DATA) \
diff -Nru gnome-pilot-0.1.54/conduits/memo_file/Makefile.am gnome-pilot-0.1.54.new/conduits/memo_file/Makefile.am
--- gnome-pilot-0.1.54/conduits/memo_file/Makefile.am	Sat Jul  1 02:58:53 2000
+++ gnome-pilot-0.1.54.new/conduits/memo_file/Makefile.am	Sun Oct 29 15:28:33 2000
@@ -49,6 +49,3 @@
 EXTRA_DIST = \
         $(ccenterConduits_DATA) \
 	$(Conduit_DATA)
-
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
