--- gnome-pilot-2.0.13/acinclude.m4	2007-01-23 11:29:37.974809889 +0200
+++ gnome-pilot-2.0.13/acinclude.m4	2007-01-23 11:37:48.065887535 +0200
@@ -52,10 +52,10 @@
 		piversion_include="pi-version.h"
 	fi
 
-	pi_version=`cat $incdir/pi-version.h|grep PILOT_LINK_VERSION|sed 's/#define PILOT_LINK_VERSION \([[0-9]]*\)/\1/'`
-	pi_major=`cat $incdir/pi-version.h|grep PILOT_LINK_MAJOR|sed 's/#define PILOT_LINK_MAJOR \([[0-9]]*\)/\1/'`
-	pi_minor=`cat $incdir/pi-version.h|grep PILOT_LINK_MINOR|sed 's/#define PILOT_LINK_MINOR \([[0-9]]*\)/\1/'`
-	pi_patch=`cat $incdir/pi-version.h|grep PILOT_LINK_PATCH|sed 's/#define PILOT_LINK_PATCH \"\(.*\)\"/\1/'`
+	pi_version=$(awk '/define PILOT_LINK_VERSION/{print [$]3}' $incdir/pi-version.h)
+	pi_major=$(awk '/define PILOT_LINK_MAJOR/{print [$]3}' $incdir/pi-version.h)
+	pi_minor=$(awk '/define PILOT_LINK_MINOR/{print [$]3}' $incdir/pi-version.h)
+	pi_patch=$(awk '/define PILOT_LINK_PATCH/{print [$]3}' $incdir/pi-version.h | tr -d '"')
 
 	PILOT_LINK_VERSION="$pi_version.$pi_major.$pi_minor$pi_patch"
 
