diff -urN aaa/gnome-pilot-2.0.13/gpilotd/Makefile.am gnome-pilot-2.0.13/gpilotd/Makefile.am
--- aaa/gnome-pilot-2.0.13/gpilotd/Makefile.am	2005-04-09 06:14:27.000000000 +0200
+++ gnome-pilot-2.0.13/gpilotd/Makefile.am	2006-08-13 22:01:25.032053000 +0200
@@ -140,7 +140,9 @@
 libgpilotdconduitincludedir = $(includedir)/gpilotd
 
 libgpilotdconduit_la_LDFLAGS = \
-	-version-info $(GPILOTD_CONDUIT_CURRENT):$(GPILOTD_CONDUIT_REVISION):$(GPILOTD_CONDUIT_AGE)
+	-version-info $(GPILOTD_CONDUIT_CURRENT):$(GPILOTD_CONDUIT_REVISION):$(GPILOTD_CONDUIT_AGE) \
+		$(GNOME_PILOT_LIBS) 			\
+		$(NULL)
 
 ###
 
--- gnome-pilot-2.0.15/libgpilotdCM/Makefile.am~	2005-09-20 00:56:21.000000000 +0200
+++ gnome-pilot-2.0.15/libgpilotdCM/Makefile.am	2007-01-21 15:36:17.745489000 +0100
@@ -43,5 +43,7 @@
 libgpilotdcm_la_LDFLAGS = \
 	-version-info $(GPILOTD_CM_CURRENT):$(GPILOTD_CM_REVISION):$(GPILOTD_CM_AGE)
 
+libgpilotdcm_la_LIBADD = $(GNOME_PILOT_LIBS)
+
 EXTRA_DIST = 		\
 	$(gobfiles)
