--- configure.in~	Mon Aug 12 09:58:11 2002
+++ configure.in	Mon Aug 12 09:59:16 2002
@@ -78,7 +78,7 @@
 
 AM_PATH_GLIB(1.2.0,,AC_MSG_ERROR(Test for GLIB failed.),[glib gmodule])
 
-PILOT_LINK_CHECK(0.9.3)
+dnl PILOT_LINK_CHECK(0.9.3)
 GOB_CHECK(1.0.4)
 
 dnl This is need by libglade
@@ -169,6 +169,7 @@
   AC_DEFINE(HAVE_SA_LEN)
 fi
 
+AM_CONDITIONAL(USING_OAF, [false])
 
 dnl Create files.
 AC_OUTPUT([
@@ -186,6 +186,7 @@
 capplet/Makefile
 utils/Makefile
 idl/Makefile
+intl/Makefile
 libgpilotdCM/Makefile
 po/Makefile.in
 ],[sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile])

--- configure.in~	Mon Aug  7 07:18:10 2000
+++ configure.in	Mon Aug 12 12:35:43 2002
@@ -111,7 +111,14 @@
 GPILOT_LIBS="-lgpilotd -lgpilotdcm -lgpilotdconduit `gnome-config --libs gnome` `glib-config --libs glib gmodule`"
 GPILOT_INCLUDEDIR="-I${includedir} `gnome-config --cflags gnome` `glib-config --cflags glib gmodule`"
 
+PISOCK_INCLUDEDIR="-I/usr/X11R6/include/libcapplet1"
+PISOCK_LIBS="-lpisock"
+BUILD_INCLUDED_LIBINTL=no
+
 dnl Automake @VARIABLE@ exports.
+AC_SUBST(PISOCK_INCLUDEDIR)
+AC_SUBST(PISOCK_LIBS)
+AC_SUBST(BUILD_INCLUDED_LIBINTL)
 AC_SUBST(CAPPLET_LIBS)
 AC_SUBST(DL_LIBS)
 AC_SUBST(GNOME_PILOT_MAJOR)
--- configure.in~	Mon Aug 12 12:52:16 2002
+++ configure.in	Mon Aug 12 12:54:28 2002
@@ -24,6 +24,8 @@
 
 GNOME_INIT(capplet applets)
 
+dnl AC_PROG_INTLTOOL([0.20])
+
 AC_ISC_POSIX
 AC_PROG_CC
 AC_PROG_CXX
