diff -Nru gnome-pilot-0.1.49/capplet/Makefile.am gnome-pilot-0.1.49.new/capplet/Makefile.am
--- gnome-pilot-0.1.49/capplet/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/capplet/Makefile.am	Sun Mar 26 15:17:17 2000
@@ -57,8 +57,8 @@
 appicon_DATA = gnome-palm.png
 
 install-data-local:
-	$(mkinstalldirs) $(PalmPilotdir)
-	$(mkinstalldirs) $(Conduitsdir)
-	$(mkinstalldirs) $(gladedir)
-	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(PalmPilotdir)/.directory
-	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(Conduitsdir)/.directory
+	$(mkinstalldirs) $(DESTDIR)$(PalmPilotdir)
+	$(mkinstalldirs) $(DESTDIR)$(Conduitsdir)
+	$(mkinstalldirs) $(DESTDIR)$(gladedir)
+	$(INSTALL_DATA) $(srcdir)/PalmPilot.directory $(DESTDIR)$(PalmPilotdir)/.directory
+	$(INSTALL_DATA) $(srcdir)/Conduits.directory $(DESTDIR)$(Conduitsdir)/.directory
diff -Nru gnome-pilot-0.1.49/conduits/backup/Makefile.am gnome-pilot-0.1.49.new/conduits/backup/Makefile.am
--- gnome-pilot-0.1.49/conduits/backup/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/conduits/backup/Makefile.am	Sun Mar 26 15:17:47 2000
@@ -49,6 +49,3 @@
 	$(Conduit_DATA) \
         $(ccenterConduits_DATA)
 
-install-data-local:
-	$(mkinstalldirs) $(Conduitdir)
-	$(mkinstalldirs) $(ccenterConduitsdir)
diff -Nru gnome-pilot-0.1.49/conduits/email/Makefile.am gnome-pilot-0.1.49.new/conduits/email/Makefile.am
--- gnome-pilot-0.1.49/conduits/email/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/conduits/email/Makefile.am	Sun Mar 26 15:17:57 2000
@@ -47,6 +47,3 @@
 EXTRA_DIST = \
 	$(ccenterConduits_DATA) \
 	$(Conduit_DATA)
-
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
diff -Nru gnome-pilot-0.1.49/conduits/expense/Makefile.am gnome-pilot-0.1.49.new/conduits/expense/Makefile.am
--- gnome-pilot-0.1.49/conduits/expense/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/conduits/expense/Makefile.am	Sun Mar 26 15:18:05 2000
@@ -47,5 +47,3 @@
 	$(ccenterConduits_DATA) \
 	$(Conduit_DATA)
 
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
diff -Nru gnome-pilot-0.1.49/conduits/file/Makefile.am gnome-pilot-0.1.49.new/conduits/file/Makefile.am
--- gnome-pilot-0.1.49/conduits/file/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/conduits/file/Makefile.am	Sun Mar 26 15:18:20 2000
@@ -71,11 +71,6 @@
 Mimedir = $(datadir)/mime-info
 Mime_DATA = palm.keys
 
-
-install-data-local:
-	$(mkinstalldirs) $(ccenterConduitsdir)
-	$(mkinstalldirs) $(Conduitdir)
-
 EXTRA_DIST = \
 	$(Conduit_DATA) \
         $(ccenterConduits_DATA) \
diff -Nru gnome-pilot-0.1.49/conduits/memo_file/Makefile.am gnome-pilot-0.1.49.new/conduits/memo_file/Makefile.am
--- gnome-pilot-0.1.49/conduits/memo_file/Makefile.am	Tue Mar 14 18:17:47 2000
+++ gnome-pilot-0.1.49.new/conduits/memo_file/Makefile.am	Sun Mar 26 15:18:29 2000
@@ -45,6 +45,3 @@
 EXTRA_DIST = \
         $(ccenterConduits_DATA) \
 	$(Conduit_DATA)
-
-install-data-local:
-	$(mkinstalldirs) $(Conduitsdir)
